app/activeplan/homepage/contactArrangements/get.controller.test.js

```diff
@@ -0,0 +1,85 @@
+const { getContactArrangements } = require('./get.controller')
+const { getSentencePlanComments } = require('../../../../common/data/sentencePlanningApi')
+
+jest.mock('../../../../common/data/sentencePlanningApi')
+
+const commentsEmpty = {}
+const commentsPresent = require('../../../../mockServer/sentencePlanComments/1.json')
+
+describe('getContactArrangements', () => {
+  const req = {
+    path: '/this/is/my/path',
+    params: {
+      planId: 1234,
+    },
+    headers: {
+      'x-auth-token': '1234',
+    },
+    body: {},
+    errors: {},
+    errorSummary: {},
+    contactArrangements: null,
+  }
+  const res = {
+    render: jest.fn(),
+  }
+
+  beforeEach(() => {
+    req.renderInfo = {}
+    delete req.body.contactArrangements
```

Comment:
delete action should be done on an 'afterEach'

Reply:
Or re-declare the req object in the beforeEach every time? That would ensure the integrity. Then renderInfo could just be in the declaration.